src/fuzz/fuzz-hidraw-id-desc.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+max_len = 4096
```

Comment:
Could you remove this file and add something like
```c
if (size > 4096)
    return 0;
```
to the fuzzer?

In the meantime, I've just opened to https://github.com/systemd/systemd/pull/13361 to get rid of all the "options" files we don't need any more.

Reply:
I added the `max_len` config mostly because I thought libFuzzer would set it to the size of the largest file in the corpus otherwise. The corpus currently consists of valid HID report descriptors which are rather small compared to their theoretical maximal length. Would you recommend to include larger samples in the corpus instead?

Reply:
On ClusterFuzz when `libFuzzer` is used as a fuzzing engine some random values between 1 and 10000 is picked and passed to the fuzzers, which kind of helps us to generate corpora with no seed corpus whatsoever :-) As far as I know it's called `random_max_len_strategy` there. I'm not sure what's going on Fuzzit (where *.options files aren't supported) though. So far I've just moved the corpora OSS-Fuzz has been accumulating for over a year to Fuzzit. @yevgenypats could you help us out? Would it make sense to pass `-max-len` to fuzzers on Fuzzit to avoid the default behavior of libFuzzer?

Anyway, usually we don't keep corpora in the repository (mostly because they are relatively large even when compressed). Instead we start with several files just to make sure that the basics are covered and eventually add testcases that have triggered any issues on ClusterFuzz, Fuzzit and so on. So I'd just leave the 2 files where they are and add a testcase crashing the fuzzer under UBsan. Hope it makes sense.

Reply:
I think `-max-len 4096` is the default behaviour anyway even in case of a large test-case. I'll validate that now.

Reply:
> I think -max-len 4096 is the default behaviour anyway even in case of a large test-case. I'll validate that now.

Is it also the default in case of a small test case? The corpora consists of files with ~65 bytes, but we would like to generate samples up to 4096 bytes.

Reply:
Per small files, yes, for sure. per large files yes that makes sense to pass `-max_len=4096`, wasn't aware of this but now tested and it is indeed uses the largest file as max_len or 4096

Reply:
Thanks for testing this! As a result, I will remove the `.options` file.